wfd/common/message_handler.h

```diff
@@ -79,6 +78,14 @@ class MessageHandler {
     assert(observer_);
   }
 
+  inline SourceMediaManager* GetSourceMediaManager() {
```

Comment:
this is a conversion function -- not a getter. ToSourceMediaManager(MediaManager* from)